tests/test_decollate.py

```diff
@@ -82,7 +73,7 @@ def test_decollation(self, _, data, batch_size=2, num_workers=2):
             transforms = Compose([LoadImaged("image"), transforms])
 
         dataset = CacheDataset(data, transforms, progress=False)
-        loader = DataLoader(dataset, batch_size=batch_size, shuffle=True, num_workers=num_workers)
+        loader = DataLoader(dataset, batch_size=batch_size, shuffle=False, num_workers=0)
```

Comment:
does the previous value cause errors?

Reply:
Shuffling should cause a problem, since it's compared with images directly from the dataset which aren't shuffled. I'll revert the `num_workers` since that doesn't cause any problems.